### DISCUSSION TITLE
played normally only 2~3 times on iPhone SE2

### DISCUSSION THREAD

**Discussion by ParkDongJo** [vimeo/player.js#1014] [Bug reports]
### Expected Behavior
play infinitly as same duration when used loop option as true

### Actual Behavior
be played as quickly speed after 2~3 times when used loop option as true

### Steps to Reproduce
I used loop option for infinitly play. but it seems like to play normally only 3 times

URL : https://player.vimeo.com/video/685704665?h=e42cb544f8&controls=0&loop=1&background=1&quality=720p&app_id=122963

** Test Device **
Device iPhone SE / iPhone SE2
OS : 14.6 / 14.5

1. open this [url](https://player.vimeo.com/video/685704665?h=e42cb544f8&controls=0&loop=1&background=1&quality=720p&app_id=122963) on safari
2. after playing from start to finish 2-3 times
3. be played as quickly speed

https://user-images.githubusercontent.com/16081624/157173304-aa088ede-8533-4d51-b89e-a4031ba1b03a.mov

**Comment by adwin-recover** [comment]
@ParkDongJo  I've faced the same issue as yours, is there any solution?
